engine/src/flutter/tools/mcp/README.md

```diff
@@ -0,0 +1,28 @@
+# Engine MCP
+
+This is an MCP server to help Gemini CLI work on the engine.
+
+It runs over stdout. The CWD is assumed to be `//engine/src/flutter`. That
+matches the CWD when executing Gemini CLI from that directory where it is set
+up.
+
+## Testing
+
+The server can be run an queried manually with the following example queries.
+Automated testing is a bit lacking until we can get it integrated with the
```

Comment:
Could you clarify what the difficulty integrating with the workspace is?

Reply:
So the "if you want to use this; you need to follow extra steps" vs gclient sync picks up the new mcp_dart package?

Reply:
This setup involves no extra steps.  Integrating it into the workspace will require the extra step of calling "gclient sync" for it to work.  We want it in the workspace at some point since that's a requirement to avoid network calls while executing tests when we want to start running the tests on ci.  This is actually a more convenient form, especially if mcp_dart is going to be changing frequently.

Reply:
Doh, our CI won't let packages exist in this state temporarily.  I'll have to migrate it before we land.